lib/util.rb

```diff
@@ -35,22 +35,18 @@ def create_pr!(repo, branch:, title:, description:)
 end
 
 def get_file_contents(repo_name, path)
-  begin
```

Comment:
This is the sort of thing we could probably catch with rubocop-govuk - could you install that at some point too please?

Reply:
Yep, good idea, I should add CI as well probably